linux/devdoc/platform_linux_requirements.md

```diff
@@ -0,0 +1,51 @@
+# platform_linux
+
+## Overview
+
+`platform_linux` provides initializes the completion_port system.
+
+## Exposed API
+
+```c
+MOCKABLE_FUNCTION(, int, platform_init);
+MOCKABLE_FUNCTION(, void, platform_deinit);
+MOCKABLE_FUNCTION(, COMPLETION_PORT_HANDLE, platform_get_completion_port);
+```
+
+### platform_init
+
+```c
+MOCKABLE_FUNCTION(, int, platform_init);
+```
+
+`platform_init` initializes the completion port object if it's not initialized
```

Comment:
>the completion port object [](http://example.com/codeflow?start=28&length=26)

So there can only ever be just 1 completion port?

Reply:
Yes.  For now.